src/fcp/v2_parser.py

```diff
@@ -453,6 +454,14 @@ def string(self, args: List[str]) -> str:
         """Parse a string node of the fcp AST."""
         return args[0].value[1:-1]  # type: ignore
 
+    def value(self, args: List[str]) -> Any:
+        """Parse a value node of the fcp AST."""
+        return args[0]
```

Comment:
shadowing the previous definition of value here: https://github.com/joajfreitas/fcp-core/pull/215/files#diff-fcbdecae9e754e698f62ca5ca0165e7d284341c94cdb50c29811eda64dc83db2L441